app/icon.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
```

Comment:
Yep, if you want to extend from it or whatnot. But isn't this the specific close icon ? (due to the `<path>`)